Library/Formula/xpwn.rb

```diff
@@ -0,0 +1,32 @@
+class Xpwn < Formula
+  homepage "https://github.com/planetbeing/xpwn"
+  url "https://github.com/planetbeing/xpwn.git"
```

Comment:
Homebrew/homebrew doesn't accept unblessed forks either, alas. If you fork it, *and* upstream bless the new fork as the official place to go for support/releases/etc that's cool - But otherwise, it'll need to come from the currently existing upstream repo & maintainer.